src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(number) {
+    return function(value) {
+      count++;
+
+      const result = value + number;
+
+      if (count % 2 === 0 && count > 3) {
```

Comment:
Consider combining the conditions in the if statement to avoid nesting ternary operators or multiple if-else statements. Since you're only checking one condition, it's not necessary to use an else statement; you can return `result` directly after the if block.